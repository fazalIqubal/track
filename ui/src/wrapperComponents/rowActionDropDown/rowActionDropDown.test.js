import React from 'react';
import ReactDOM from 'react-dom';
import {configure, shallow } from 'enzyme';
import { shallowToJson} from 'enzyme-to-json';
import RowActionDropdown from './rowActionDropDown'
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('should render correctly', () => {
    let rowDropDown;
    beforeEach(() => {
        rowDropDown = shallow(<RowActionDropdown/>
        );
    });
    afterEach(() => {
        rowDropDown = null;
    });

    it('should render correctly ', () => {
        expect(shallowToJson(rowDropDown)).toMatchSnapshot()
    });

    it('contains an <Space/> component', () => {
        expect(rowDropDown.find('Space').exists()).toEqual(true);
    });

    it('should have <Dropdown/>', () => {
        expect(rowDropDown.find('Dropdown').exists()).toEqual(true);
    });
})