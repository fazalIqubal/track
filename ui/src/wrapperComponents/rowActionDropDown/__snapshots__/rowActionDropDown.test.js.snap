// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly should render correctly  1`] = `
<Space
  size="middle"
>
  <Dropdown
    mouseEnterDelay={0.15}
    mouseLeaveDelay={0.1}
    overlay={
      <Menu>
        <MenuItem>
          Edit
        </MenuItem>
        <MenuItem>
          Delete
        </MenuItem>
      </Menu>
    }
  >
    <a
      className="ant-dropdown-link"
      onClick={[Function]}
    >
      <ForwardRef(EllipsisOutlined) />
    </a>
  </Dropdown>
</Space>
`;
